### DISCUSSION TITLE
Is the demo still working?

### DISCUSSION THREAD

**Discussion by imborxcom** [srcnalt/3d-profile-avatars#12] [Q&A] [answered]
I tried to check the demo (https://codesandbox.io/s/avatar-view-wrn2u) and unfortunately it says:

"Error
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `App`"

<img width="1436" alt="Screenshot 2023-11-29 at 16 06 46" src="https://github.com/srcnalt/3d-profile-avatars/assets/50377375/fa76fa75-6d8c-494c-8bfb-ba39b3bdcb8f">

**Comment by srcnalt** [comment, marked as answer]
Hi, it was not updated with the latest package. Just fixed it, can you try and check again?
